lib/Basic/Platform.cpp

```diff
@@ -431,147 +431,139 @@ llvm::Triple swift::getUnversionedTriple(const llvm::Triple &triple) {
                       unversionedOSName);
 }
 
+namespace {
+
+enum class Platform {
+  macOS,
+  iOS,
```

Comment:
Is tvOS intentionally excluded because it never diverges from iOS?

Reply:
Yeah, that assumption is already pervasive in `RuntimeVersions.def`.